Source/BackgroundCallNotifications.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  BackgroundCallNotifications.swift
+//  SparkSDK
+//
+//  Created by Rik van den Brule on 29/11/2016.
+//  Copyright © 2016 Cisco. All rights reserved.
+//
+
+import Foundation
+
+/// Defines notifications to notify SparkSDK of calls when the application is in the background.
+/// 
+/// These notifications can be posted to `NSNotificationCenter` to let SparkSDK handle background calls.
+@objc public class SparkBackgroundCallNotifications: NSObject {
```

Comment:
I agree that this is more Swifty. I implemented it this way because we needed to access these notifications in Objective C parts of our app and this is not possible directly with your suggested implementation I think. This way I did not have to introduce the name of the notification at two points in the codebase.

I realise that SparkSDK is a pure Swift library and it currently does not include interfaces to call the SDK directly from Objective C code. It would be valuable to add a bridging interface I would think, but if you'd rather keep the project Swift only for now and introduce such an interface later I don't mind to rewrite the code as you suggest.